src/components/Tooltip/Tooltip.styles.ts

```diff
@@ -105,7 +105,10 @@ const DivStyledTooltipParent = styled.div`
     ${resetCSS}
     color: #252525;
     position: relative;
-    &:hover > div:nth-child(2) {
+`;
+
+const DivStyledTooltipContent = styled.div`
+    &:hover + div {
```

Comment:
oh sure.
Done

Reply:
Great job! 
Please take a look at this, sorry I combined both reviews in one

> This is a little bug where the cursor sees the hidden text see the this video https://user-images.githubusercontent.com/13832392/171008137-44e377a6-bb79-4015-9104-25fb207a1861.mov
> 
> this could be easily solved by adding `visibility:hidden` and `visibility:visible` along side with the opacity